Description: Change the path to binson.simple to debian installation
Forwarded: not-needed
Author: Floris Bruynooghe <flub@devork.be>

--- a/pic2plot/gram.cc
+++ b/pic2plot/gram.cc
@@ -1041,7 +1041,7 @@
    120,   121,   122,   123,    -1,   125
 };
 /* -*-C-*-  Note some compilers choke on comments on `#line' lines.  */
-#line 3 "/usr/local/share/bison.simple"
+#line 3 "/usr/share/bison/bison.simple"
 /* This file comes from bison-1.28.  */
 
 /* Skeleton output parser for bison,
@@ -1255,7 +1255,7 @@
 #endif
 #endif
 
-#line 217 "/usr/local/share/bison.simple"
+#line 217 "/usr/share/bison/bison.simple"
 
 /* The user can define YYPARSE_PARAM as the name of an argument to be passed
    into yyparse.  The argument should have type void *.
@@ -3227,7 +3227,7 @@
     break;}
 }
    /* the action file gets copied in in place of this dollarsign */
-#line 543 "/usr/local/share/bison.simple"
+#line 543 "/usr/share/bison/bison.simple"
 
   yyvsp -= yylen;
   yyssp -= yylen;
